app/src/main/res/values/styles.xml

```diff
@@ -468,6 +468,28 @@
     <item name="android:textAlignment">viewStart</item>
   </style>
 
+  <style name="AddMultiProfileHeading" parent="TextView.Common1">
+    <item name="android:textAlignment">viewStart</item>
+    <item name="fontFamily">sans-serif-medium</item>
+    <item name="android:textColor">@color/color_def_white</item>
+    <item name="android:textSize">14sp</item>
+  </style>
+
+  <style name="AddMultiProfileBody" parent="TextView.Common1">
```

Comment:
Please not add new style `AddMultiProfileBody` instead of `AddMultiProfileBody` use `TextFieldLabel` and add `android:textColor="@color/component_color_shared_secondary_4_text_color"` in `TextView`

Avoid using colors from `color_def` use color from `component_color_shared`